en/SUMMARY.md

```diff
@@ -23,13 +23,13 @@
    * [Sensors](SetupView/Sensors.md)
    * [Flight Modes](SetupView/FlightModes.md)
    * [Power](SetupView/Power.md)
-   * [Motors](SetupView/Motors.md)
+   * [Motors (ArduSub)](SetupView/Motors.md)
    * [Safety](SetupView/Safety.md)
      * [Safety (ArduPilot)](SetupView/safety_ardupilot.md)
    * [Tuning](SetupView/Tuning.md)
      * [Tuning (ArduPilot)](SetupView/tuning_ardupilot.md)
-       * [ArduCopter](SetupView/tuning_arducopter.md)
```

Comment:
FYI, the previous format is better from a browsing point of view, but this is also what is shown as heading in search results - so much worse for search.